rbc/npy_mathimpl.py

```diff
@@ -1,5 +1,39 @@
 import numpy as np
-from numba.targets import mathimpl
+from numba.targets import mathimpl, ufunc_db
 
+# tell numba to wire np.exp2 to libm exp2.
 mathimpl.unary_math_extern(np.exp2, "exp2f", "exp2")
 mathimpl.unary_math_extern(np.log2, "log2f", "log2")
+
+
+def np_logaddexpr_impl(context, builder, sig, args):
+    def impl(x, y):
+        a = np.exp(x)
+        b = np.exp(y)
+        return np.log(a + b)
+
+    return context.compile_internal(builder, impl, sig, args)
+
+
+def np_logaddexpr2_impl(context, builder, sig, args):
+    def impl(x, y):
+        a = np.power(2, x)
+        b = np.power(2, y)
+        return np.log2(a + b)
+
+    return context.compile_internal(builder, impl, sig, args)
```

Comment:
This will likely not numerically replicate the functions described. See reference https://github.com/numpy/numpy/blob/maintenance/1.18.x/numpy/core/src/npymath/npy_math_internal.h.src#L585-L625

Reply:
Hey @stuartarchibald, my bad for not looking at the current implementation. Thanks for the tip.